src/Scene/NodeProcess.js

```diff
@@ -176,16 +183,17 @@ NodeProcess.prototype.refineNodeLayers = function(node, camera, params) {
         updateNodeImagery
     ];
 
-    for (let i=0; i<2; i++) {
-        if (node.pendingLayers[i] === undefined
-            && (!node.loaded || node.downScaledLayer(i))) {
+    // WARNING don't use declaration function to reset flag
+    // var resetPendingFlag = function(id){node.pendingLayers[id] = undefined;};
 
-            node.pendingLayers[i] = true;
+    for (let typeLayer=0; typeLayer<2; typeLayer++) {
```

Comment:
It's probably a good call to rename `i` -> `layerType` but it could probably be defered to a specific cleanup PR.